src/components/select/select-list/select-list.component.js

```diff
@@ -66,6 +71,45 @@ const SelectList = React.forwardRef(
     const listActionButtonRef = useRef();
     const { blockScroll, allowScroll } = useScrollBlock();
 
+    // small adjustment of react-virtual's default observeElementRect in order to account for the table header in multicolumn mode
+    // (the only change from the default is subtracting the HEIGHT_ADJUSTMENT from the rect's height)
+    /* istanbul ignore next */
```

Comment:
Yes I wasn't comfortable ignoring the whole thing either - but then I decided it was just basically testing the third-party package. On the other hand, I've had to override this for our purposes so we really should test that. I don't see how I can do a sensible unit test for this but I'll see if I can add a cypress test to ensure that when navigating with the keyboard the selected option is always visible (because that's the issue I was fixing)